ee/codegen/src/generators/nodes/final-output-node.ts

```diff
@@ -42,12 +43,19 @@ export class FinalOutputNode extends BaseSingleFileNode<
       ],
     });
 
-    const nodeInput = this.getNodeInputByName("node_input");
```

Comment:
[nit] let's fallback to this logic in case `this.nodeData.outputs` is null/undefined/empty for whatever reason. In the future once we have confidence that we've migrated all nodes as generic/base nodes, we can axe.